modules/support/client/components/BlockMemberLink.component.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * A link to beport member
+ * @param {string} username - the username of the user to report
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import '@/config/client/i18n';
+import { useTranslation } from 'react-i18next';
+
+import * as api from '../api/block.api';
+
+
+
+
+export default function BlockMemberLink({ username }) {
+  const { t } = useTranslation('support');
+
+  async function handleBlockMember({ username }) {
+    const data = await api.block(username); 
+  }
+
+  return (
+    <a
+      href={`/support?report=${username}`}
+      className="small text-muted"
+      onClick={() => handleBlockMember(username)}
+      aria-label={t('Block member {{username}} to support', { username })}
```

Comment:
This line is for blind keyboard users. Mentioning support doesn't make sense here, how about just `Block member {{username}}`?